Fabric.Identity.API/Quickstart/Account/AccountController.cs

```diff
@@ -226,87 +232,52 @@ public async Task<IActionResult> ExternalLoginCallback(string returnUrl)
             //read external identity from the temporary cookie
             var info = await HttpContext.Authentication.GetAuthenticateInfoAsync(
                            IdentityServerConstants.ExternalCookieAuthenticationScheme);
-            var tempUser = info?.Principal;
-            if (tempUser == null)
-            {
-                throw new Exception("External authentication error");
-            }
-
-            //retrieve claims of the external user
-            var claims = tempUser.Claims.ToList();
-
-            //try to determine the unique id of the external user - the most common claim type for that are the sub claim and the NameIdentifier
-            //depending on the external provider, some other claim type might be used
-            var userIdClaim = claims.FirstOrDefault(x => x.Type == JwtClaimTypes.Subject);
-            if (userIdClaim == null)
-            {
-                userIdClaim = claims.FirstOrDefault(x => x.Type == ClaimTypes.NameIdentifier);
-            }
-
-            if (userIdClaim == null)
-            {
-                throw new MissingUserClaimException(ExceptionMessageResources.MissingUserClaimMessage);
-            }
-
             var schemaItem = info.Properties.Items.FirstOrDefault(i => i.Key == "scheme");
```

Comment:
I know it's existing code, but would it be Schem**_e_** Item?

Reply:
probably